### ISSUE TITLE
What's wrong within LoRA datasets?

### ISSUE THREAD

**Issue by wolo-wolo** [xaCheng1996/DiFF#4]
After unzip LoRA and get:
/test/I2I/__MACOSX/LoRA
/test/I2I/LoRA

However, the image seems to have a format error. Could you check and provided correct version?
Thks

**Comment by xaCheng1996** [comment]
Hi,
You can just delete the __MACOSX. This folder is an auto-generated directory that is used only in MAC OS.

**Comment by wolo-wolo** [comment]
> Hi, You can just delete the __MACOSX. This folder is an auto-generated directory that is used only in MAC OS.

Thanks for your quick reply. I found images that start with the '._,' prefix, e.g., /test/I2I/LoRA/id_0224/._lora_001_000.png, are less than 4kB in size, which cannot be read and visualized. I wonder if I should just delete those files.

Another question: 
Table 7 in the paper provides four detection methods trained on FF++ and reports AUC generalized to T2I I2I FS FE in test subdatasets. To benchmark this table in my current work, I need to strictly follow its settings. Can you tell me:
1) When using FF++ for training, do you follow its official 820 videos as a training set? How many frames are extracted from each video, and how are faces extracted?
2) As for T2I I2I FS FE test data, it seems that the downloaded test set only has forged images. Are there no real images in the test subset? Or is it convenient for you to provide them?

**Comment by wolo-wolo** [comment]
@xaCheng1996 How do we reproduce the results in Table 7, test in DiFF(w/o crop face?) w.r.t. train on FF++(compression? frame sample ratio? face extraction)?

**Comment by hengrui0516** [comment]
@wolo-wolo Hi, I have the same question about "As for T2I I2I FS FE test data, it seems that the downloaded test set only has forged images. Are there no real images in the test subset? Or is it convenient for you to provide them?", I wonder whether you find the answer?

**Comment by wolo-wolo** [comment]
> [@wolo-wolo](https://github.com/wolo-wolo) Hi, I have the same question about "As for T2I I2I FS FE test data, it seems that the downloaded test set only has forged images. Are there no real images in the test subset? Or is it convenient for you to provide them?", I wonder whether you find the answer?

Plz see Dataset Download in README.MD
“For Pristine images and prompts, please fill this [google form](https://forms.gle/qRM1iZMLSTEymczN7) for the accessment.”

**Comment by hengrui0516** [comment]
@wolo-wolo Thx，I've download the real samples through the link. But I wonder how AUCs of T2I I2I FS FE subsets are calculated (in Table 7 of the paper)? As far as I know, AUC can't be calculated if test set consists of all fake samples and no real ones? Do you have any idea?
